test-jest/chart/LineChart.spec.tsx

```diff
@@ -84,6 +79,11 @@ describe('<LineChart />', () => {
     const curves = container.querySelectorAll('.recharts-line .recharts-line-curve');
     expect(curves).toHaveLength(1);
     const path = curves[0].getAttribute('d');
+
+    if (!path) {
+      throw new Error('Path is null');
+    }
```

Comment:
nit: all of these types of errors could be negative assertions `.toBeNull`, etc.

fine for now

Reply:
yes but you will have to deal with typescript too

![image](https://user-images.githubusercontent.com/8848006/211257695-15a723ab-8404-43d1-8f7e-2dd204b136bb.png)

So you have to do use a lot of null coalescing and optional chaining, I find it more elegant to just throw an error.

```javascript
const path = curves[0].getAttribute('d');

  /* if (!path) {
    throw new Error('Path is null');
  } */

  expect(path).not.toBeNull();

  expect((path?.length ?? 0) - (path?.split('M')?.join('')?.length ?? 0)).toEqual(2);
```

Reply:
Eh to each their own. I would rather use one non-null assertion (!) that it will fail on if it's null (for tests only of course) than optional chaining or throwing an error. It's style so all good